app/waf/src/repositories/common.ts

```diff
@@ -15,11 +24,80 @@ export class CommonRepository<
     return super.create(data, options);
   }
 
-  updateById(id: ID, data: DataObject<T>, options?: Options): Promise<void> {
-    Object.assign(data, {
-      updatedAt: new Date().toISOString(),
-    });
+  find(filter?: Filter<T>, options?: Options): Promise<T[]> {
+    if (options !== undefined && options['tenantId']) {
+      let tenantFilter: Filter = {where: {tenantId: options['tenantId']}};
+      return super.find(tenantFilter as (Filter<T>));
+    } else {
+      return super.find(filter, options);
+    }
+  }
+
+  async findById(id: ID, filter?: Filter<T>, options?: Options): Promise<T> {
+    if (options !== undefined && options['tenantId']) {
```

Comment:
Is it safe to use options parameter to pass tenantId? And in the 1st if clause, the options parameter is suppressed, so that its behavior will be different from the else clause technically. Is there a better way to merge these two clause?